tray/monitor.go

```diff
@@ -241,6 +241,10 @@ func (ti *Instance) pollingMonitor() {
 	fullUpdate := true
 	fullUpdateLast := time.Time{}
 	for {
+		if ti.state.notificationsStatus == Invalid {
+			ti.updateSettings()
+		}
+
```

Comment:
I don't think these lines are needed. Until user logs in, all interval ticker updates are "full updates", so they fetch settings during every loop anyway (line 252). And it makes no sense to try to fetch settings before ping, when daemon is not available.

Reply:
right, missed that they are full updates. I'll remove it.